srcpkgs/kotlin/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'kotlin'
+pkgname=kotlin
+version=1.2.51
+revision=1
+wrksrc="${pkgname}c"
+only_for_archs="i386 x86_64"
+makedepends="unzip setconf"
+depends="openjdk"
+short_desc="Statically typed programming language that can interoperate with Java"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="Apache"
```

Comment:
please use spdx identifier, should be Apache-2.0

Reply:
Alright. Thanks!